recipes/unifhy/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "unifhy" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/unifhy-org/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d219ad33ea94f78aaddc9757fadc0793f008dee33d5ebbca4e32c0b42fbf9170
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - cf-python >=3.11.0
+    - cftime
+    - cfunits
+    - esmpy
+    - netcdf4 >=1.5
+    - numpy >=1.16
+    - pyproj >=3.0
+    - python >=3.7
+    - pyyaml >=5.3
+
+test:
+  imports:
+    - unifhy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/unifhy/
+  summary: Unified Framework for Hydrology
+  description: |
+    The Python package unifhy (Unified Framework for Hydrology) is a hydrological modelling framework 
+    which combines interchangeable modelling components for the surface layer, subsurface, and open water 
+    parts of the terrestrial water cycle. It is designed to foster collaborations between land surface, 
+    hydrological, and groundwater modelling communities.
+  license: BSD-3-Clause-Clear
```

This is just normal BSD 3-clause (you can see the [clear version](https://spdx.org/licenses/BSD-3-Clause-Clear.html) if you like)
```suggestion
  license: BSD-3-Clause
```

---

Thanks @dopplershift, I was wondering why grayskull added clear. Now I know it was a mistake. :-)